apps/protocol-frontend/src/pages/Signature.tsx

```diff
@@ -0,0 +1,12 @@
+import { Box } from '@chakra-ui/react';
+import SignatureLayout from '../components/SignatureLayout';
+
+const Signature = () => {
```

Comment:
We should definitely rename this to something a little more specific to this flow.

Reply:
Good call. SignatureJoinFlow? SignatureDiscordFlow ? Something like that

Reply:
Went with DiscordSignature and DiscordSignatureLayout for now. We can chat about this briefly at standup but agree should be clearer